docs/examples/datadog_exporter/datadog_exporter.py

```diff
@@ -31,6 +32,21 @@
 span_processor = DatadogExportSpanProcessor(exporter)
 trace.get_tracer_provider().add_span_processor(span_processor)
 
+# add Datadog format alongside the default propagators if already set
```

Comment:
maybe this can be in a different example? I feel like it's fine for most people to just see an example of full DD compatibility.

Reply:
This makes good sense. I'll add this and update this PR.

Reply:
@toumorokoshi I moved this as part of the distributed trace example. Basically, give developers an example of distributed trace of an OpenTelemetry instrumented client requesting from an OpenTelemetry instrumented server as well as a distributed trace of a Datadog instrumented client requesting from the same OpenTelemetry instrumented server.